src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/courtreturns/ConfirmCourtReturnResponse.kt

```diff
@@ -5,4 +5,6 @@ import com.fasterxml.jackson.annotation.JsonInclude
 @JsonInclude(JsonInclude.Include.NON_NULL)
 data class ConfirmCourtReturnResponse(
   val prisonNumber: String,
+  val location: String?,
+  val bookingId: Long
```

Comment:
these response objects should have swagger annotations

Reply:
done